src/scala/com/github/johnynek/bazel_deps/MakeDeps.scala

```diff
@@ -110,12 +110,13 @@ object MakeDeps {
         lazy val coursierResolver =
           new CoursierResolver(model.getOptions.getResolvers, ec, 3600.seconds, resolverCachePath)
 
-        resolve(
```

Comment:
note, this branch (unlike the two above) does not call this.resolve, it just calls resolve on the resolver.

See the code here:

https://github.com/ianoc/bazel-deps/blob/0789aa835604824ea37dcee6a7012710f63cc68b/src/scala/com/github/johnynek/bazel_deps/MakeDeps.scala#L108